pkg/sql/opt/optbuilder/export.go

```diff
@@ -19,9 +19,6 @@ import (
 
 // buildExport builds an EXPORT statement.
 func (b *Builder) buildExport(export *tree.Export, inScope *scope) (outScope *scope) {
-	if err := b.catalog.RequireAdminRole(b.ctx, "EXPORT"); err != nil {
```

Comment:
Can we not make this just gated on the nature of the URI?

Reply:
Yeah that's exactly what I wanted to do but there's a not so easily resolveable dep cycle as mentioned in the PR desc. If we want to reuse the URI method which is used by import, backup etc. It'll involve moving a bunch of stuff around in cloudimpl. I could dig into it if we feel strongly about this.

Reply:
oh nevermind, i think it might not involve moving too much stuff. Let me give it a shot.